### PR TITLE
change 'outrigger' references to 'juttle service'

### PR DESCRIPTION
**Pull request by go-oleg**
Resolves https://github.com/juttle/juttle-client-library/issues/52.

@mnibecker

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ import View from './view';
 import JuttleServiceHttp from './utils/http-api';
 import Errors from './errors';
 
-export default function Juttle(outriggerUrl) {
-    this.outriggerUrl = outriggerUrl;
-    this.api = new JuttleServiceHttp(`http://${outriggerUrl}`);
+export default function Juttle(juttleServiceUrl) {
+    this.juttleServiceUrl = juttleServiceUrl;
+    this.api = new JuttleServiceHttp(`http://${juttleServiceUrl}`);
```
Comment (mnibecker) [inline src/index.js]: technically this should prolly be juttleServiceHost, but whateves

**Comment by mnibecker** [comment]
lgtm

File: src/index.js (hunk 1 above)
Comment (go-oleg) [inline src/index.js]: true

Comment (go-oleg) [inline src/index.js]: gonna leave as is for now

**Pull request merged**